server/locale/FR/styles/flex/ratio--1x1.css

```diff
@@ -0,0 +1,47 @@
+.message__disclaimer span.multi:nth-of-type(1) {
+    display: none;
+}
+
+.message__headline span.multi:nth-of-type(2) {
+    display: none;
+}
+
+.message__logo-container {
+    width: 100%;
+    margin-bottom: 5%;
+}
+
+.message__logo:nth-of-type(1) {
+    width: 29px;
+    max-width: 15%;
+    margin-right: 0px;
+}
+
+.message__logo:nth-of-type(2) {
+    width: 91px;
+    max-width: 45%;
+    margin-left: 3%;
+}
+
+.message__headline {
+    font-size: 10vw;
+    line-height: 1.55em;
+}
+
+@media (min-width: 140px) {
```

Comment:
Above this media query we can add the other media query that was removed from the gpl mutation file (with a small tweak to font size): 

```
@media screen and (max-width: 240px) { 
    .message__sub-headline { 
        font-size: 0.8rem; 
    } 
}
```